improver/utilities/spatial.py

```diff
@@ -288,23 +361,28 @@ def process(self, cube):
 
         Returns:
             (tuple): tuple containing:
-                **diff_along_y_cube** (iris.cube.Cube):
-                    Cube after the differences have been calculated along the
+                **gradient_along_y_cube** (iris.cube.Cube):
```

Comment:
The docstring and code are inconsistent here - the docstring has y first, but the code has x first in the for loop on line 373. Suggest checking which order is intended.

Reply:
Resolved in f0091f6